### ISSUE TITLE
Include illustrative image(s)

### ISSUE THREAD

**Issue by nichtich** [gbv/wikidata-jskos#57]
[P18](https://www.wikidata.org/wiki/Property:P18) links to images on Wikimedia Commons. Image URLs should be returned in JSKOS field `depiction`. Image URLs can be generated from Commons file name with preferred image size like this:

http://commons.wikimedia.org/wiki/Special:FilePath/Honeycrisp.jpg?width=300px
=> https://upload.wikimedia.org/wikipedia/commons/thumb/f/f4/Honeycrisp.jpg/300px-Honeycrisp.jpg

Open question is how to comply with license. JSKOS format does not include license information so how should clients (especially Cocoda) handle it?

=> https://commons.wikimedia.org/wiki/File:Honeycrisp.jpg
=> [Jonathunder](https://commons.wikimedia.org/wiki/File:Honeycrisp.jpg) / [CC BY-SA](https://creativecommons.org/licenses/by-sa/3.0)

**Comment by nichtich** [comment]
Unfortunately Wikimedia Commons [does not support IIIF API](https://commons.wikimedia.org/wiki/Commons:International_Image_Interoperability_Framework#IIIF_and_Wikimedia_projects)

**Comment by nichtich** [comment]
Apparently the `depiction` field is already filled but the license issue remains.

**Comment by stefandesu** [comment]
Instead of return image URLs as string, couldn't we return them as an object which includes the URL as well as the license?

**Comment by nichtich** [comment]
Yes, that's the idea but it turns out complicated which fields to include in the image object without inventing your own standard. One option so far is [IIIF Presentation API](https://iiif.io/api/presentation/3.0/#rights) or [IIIF Image Info API](https://iiif.io/api/image/3.0/#56-rights).

**Comment by nichtich** [comment]
Closed in favour of #91
